docs/network-setup.md

```diff
@@ -43,6 +43,46 @@ those rules before starting.
 sudo iptables-save > iptables.rules.old
 ```
 
+*Advanced:*  In case you want to use a bridge to communicate with the microVM,
+you need to do the following additional setup:
```

Comment:
A part of this steps don't look like additional setup. They overlap with the basic tap setup. For example creating the tap interface, setting it up, etc.